lib/todo/command_menu_formatter.ex

```diff
@@ -0,0 +1,18 @@
+defmodule ToDo.CommandMenuFormatter do
```

Comment:
I actually prefer `MenuOptionsFormatter` more than the new name. New name seems/sounds unclear what it is.